backends/p4tools/modules/testgen/targets/bmv2/test_backend/common.h

```diff
@@ -0,0 +1,43 @@
+#ifndef BACKENDS_P4TOOLS_MODULES_TESTGEN_TARGETS_BMV2_TEST_BACKEND_COMMON_H_
+#define BACKENDS_P4TOOLS_MODULES_TESTGEN_TARGETS_BMV2_TEST_BACKEND_COMMON_H_
+
+#include <filesystem>
+#include <optional>
+#include <vector>
+
+#include <inja/inja.hpp>
+
+#include "backends/p4tools/modules/testgen/lib/test_object.h"
+#include "backends/p4tools/modules/testgen/lib/test_spec.h"
+#include "backends/p4tools/modules/testgen/lib/tf.h"
+
+namespace P4Tools::P4Testgen::Bmv2 {
+
+class Bmv2TF : public TF {
```

Comment:
I'd write some comment clarifying that TF is TestFramework here... You could even spell it out in the name (which would be the usual google approach).

Reply:
I can file an issue for this and expand `TF` in a separate PR across all open-source back ends? Should be a quick change.

Reply:
SGTM

Reply:
https://github.com/p4lang/p4c/issues/4229